lnbits/core/views/api.py

```diff
@@ -606,7 +626,7 @@ class ConversionData(BaseModel):
 async def api_fiat_as_sats(data: ConversionData):
     output = {}
     if data.from_ == "sat":
-        output["sats"] = int(data.amount)
+        output["sats"] = data.amount
```

Comment:
You don't cast the amount here. Why?

Reply:
mypy complains, if we set amount to float in ConversionData, then cast it to int and afterwards using it as float again.

Reply:
I have the feeling that something else might break because of this.

Reply:
can we somehow write a test for this? i'll take a look

Reply:
apparently just changing the order fixed the mypy error :) 36cb1eb4